### PR TITLE
emit_rusage only on debug mode

### PR DESCRIPTION
**Pull request by chunseoklee**
`emit_rusage` causes severe overhead on measurement. Let's enable this on debug build

ONE-DCO-1.0-Signed-off-by: Chunseok Lee <chunseok.lee@samsung.com>

### CODE REVIEW

File: runtime/onert/core/src/util/EventCollector.cc [hunk 1]
```diff
@@ -99,6 +99,9 @@ void EventCollector::onEvent(const Event &event)
       break;
   }
 
-  // Trace resource usage per each event notification
+// TODO: Add resurece measurement(e.g. RSS)
+// when ready with low overhead in release build
+#ifdef DEBUG
```
Comment (hseok-oh) [inline runtime/onert/core/src/util/EventCollector.cc:104]: It will make unused function warning on clang release build.

Comment (chunseoklee) [reply runtime/onert/core/src/util/EventCollector.cc:104]: inline does not make it work on release mode ? I will test on android build

Comment (hseok-oh) [reply runtime/onert/core/src/util/EventCollector.cc:104]: @chunseoklee I've checked and got build error

```
/home/nfs/git/ONE/runtime/onert/core/src/util/EventCollector.cc:57:13: error: unused function 'emit_rusage'
      [-Werror,-Wunused-function]
inline void emit_rusage(EventRecorder *rec, const std::string &ts)
            ^
1 error generated.
runtime/onert/core/CMakeFiles/onert_core.dir/build.make:2519: recipe for target 'runtime/onert/core/CMakeFiles/onert_core.dir/src/util/EventCollector.cc.o' failed
make[3]: *** [runtime/onert/core/CMakeFiles/onert_core.dir/src/util/EventCollector.cc.o] Error 1
```

Comment (chunseoklee) [reply runtime/onert/core/src/util/EventCollector.cc:106]: @hseok-oh I have added commit which declare emit_rusage on debug build.

**Pull request merged**